### ISSUE TITLE
Favicons

### ISSUE THREAD

**Issue by shaunanoordin** [zooniverse/Brand#23]
## Branding Improvements

We need some standardised favicons in our Brand repo.

1. At the moment, we don't have any "official" brand-approved favicons to use for our websites, notably our Custom Front Ends. (See [Zoo-Notes](https://github.com/zooniverse/zoo-notes), [Alice](https://github.com/zooniverse/text-editor), etc).
2. These CFEs have been copying the favicon from zooniverse.org to make do.
3. Problem is, even [our main website](https://www.zooniverse.org/) is using _a really old favicon._

Behold: https://static.zooniverse.org/www.zooniverse.org/favicon.ico (32x32 and 16x16 resolution)
- Our current favicon uses the older Zooniverse logo which has a far thinner bar
- It also uses ANOTHER kind of teal - #319cae! Heretic! 🔥 
  - (Yes, that's a _second_ heathen teal, after #007482 was detected on our logo files, instead of the sanctioned #00979d)

### Proposal

I propose we add some official favicons to the Brand repo. I've already whipped up 3 different favicons we could use, but these sorta things should best go under the eye of a proper designer.

Attached file: [shauns-favicons-20200921.zip](https://github.com/zooniverse/Brand/files/5255262/shauns-favicons-20200921.zip)
- contains proposed white, black, teal (#00979d) favicons with baked-in 16x16, 24x24, 32x32, 64x64 resolutions... though they each clock in at 24kb.
- contains our current [Zooniverse.org favicon](https://static.zooniverse.org/www.zooniverse.org/favicon.ico) for comparison

### Trivia

Q: where the hell did #319cae come from?
A: [ASTEROID ZOO](https://github.com/zooniverse/Asteroid-Zoo/blob/808c5cf686c88c4788286eecf537afccae05f2cd/css/common.styl#L9)

### Status

Low priority, I think? But I'm sure knowing that a million browsers out there have #319cae Zooniverse logos in their bookmarks will absolutely drive certain people crazy.

**Comment by beckyrother** [comment]
🚨  CAN'T UNSEE 🚨 

Thank you for making these Shaun! For consistency let's use the teal one (with the CORRECT TEAL 🤦‍♀️  ) everywhere; are you able to add a favicon folder in the [logos ](https://github.com/zooniverse/Brand/tree/master/style%20guide/logos) folder?
